src/diffusers/models/unet_2d_condition_flax.py

```diff
@@ -127,7 +132,17 @@ def init_weights(self, rng: jax.random.KeyArray) -> FrozenDict:
         params_rng, dropout_rng = jax.random.split(rng)
         rngs = {"params": params_rng, "dropout": dropout_rng}
 
-        return self.init(rngs, sample, timesteps, encoder_hidden_states)["params"]
+        added_cond_kwargs = None
+        if self.addition_embed_type == "text_time":
+            # TODO: how to get this from the config? It's no longer cross_attention_dim
+            text_embeds_dim = 1280
```

Comment:
You can get it from here: https://huggingface.co/laion/CLIP-ViT-bigG-14-laion2B-39B-b160k/blob/8c7a3583335de4dba1b07182dbf81c75137ce67b/config.json#L9

Reply:
so it should be `self.text_encoder.config.projection_dim`

Reply:
You are right, thanks!

Reply:
@pcuenca can we correct this?